examples/Ep_spacing.py

```diff
@@ -1,14 +1,17 @@
 from pyspecdata import *
-from SpinCore_pp.power_helper import Ep_spacing_from_phalf 
-from SpinCore_pp.config_parser_fn import configuration 
+from SpinCore_pp.power_helper import Ep_spacing_from_phalf
+from SpinCore_pp.config_parser_fn import configuration
 import matplotlib.pyplot as plt
+
 config_dict = configuration("active.ini")
 with figlist_var() as fl:
-    Ep_powers = Ep_spacing_from_phalf(est_phalf = config_dict['guessed_phalf'], 
-            max_power = config_dict['max_power'],
-            p_steps = config_dict['power_steps'],fl=fl)
-    plt.title('Evenly spaced Ep curve')
-    plt.ylabel('Signal')
-    plt.xlabel('Power / W')
+    Ep_powers = Ep_spacing_from_phalf(
+        est_phalf=config_dict["guessed_phalf"] / 4,
+        max_power=config_dict["max_power"],
+        p_steps=config_dict["power_steps"],
+        fl=fl,
+    )
+    plt.title("Evenly spaced Ep curve")
+    plt.ylabel("Signal")
+    plt.xlabel("Power / W")
```

Comment:
I just ran black here.

**But** can you please add the min dBm steps parameter as a parameter that lives inside the ODNP block (since it depends on the overall setup).  It should be included in the default active.ini, I think.

Reply:
(it can be 1.0 vs. 0.5 vs. 0.1 depending on the source being used)